src/main/java/org/codeforamerica/formflowstarter/ScreenController.java

```diff
@@ -187,7 +187,7 @@ ModelAndView postNewSubflow(
 		}
 
 		UUID uuid = UUID.randomUUID();
-		formDataSubmission.put(String.format("%sUuid",subflowName), uuid);
```

Comment:
I was prepending the subflow name to this uuid thinking that it should be unqiue to each subflow just in case we get into a situation where we're in two subflows or next to two different subflows or something like that. Though now looking at it I'm not sure that's needed

Reply:
That's fair, I was thinking because we are access the subflow by it's name as the key, we would know which subflows uuid we were accessin